src/bundle/Resources/views/themes/admin/trash/list.html.twig

```diff
@@ -50,7 +50,7 @@
                         <div class="ibexa-trash-search-form__input-wrapper">
                             <input
                                 type="text"
-                                class="form-control ibexa-trash-search-form__input"
+                                class="form-control ibexa-input ibexa-input--text ibexa-trash-search-form__input"
```

Comment:
Can some of `bexa-trash-search-form__input` styles be deleted given `ibexa-input ibexa-input--text` was added?

Reply:
i think this input was not styled at all by this class.